.circleci/config.yml

```
@@ -129,6 +129,9 @@ jobs:
           paths:
             - ./packages/.artifacts
             - ./build
+      - run:
+          name: Prepare CircleCI artifacts
+          command: PYODIDE_BASE_URL="$CIRCLE_BUILD_URL/artifacts/0/root/repo/build/" make build/console.html
```

I think it will work if you do:
```suggestion
          command: PYODIDE_BASE_URL="./" make build/console.html
```

---

Yes of course that's the most straightforward fix. Thanks!